### PR TITLE
Add Support CSS Modules

### PR DESCRIPTION
**Pull request by ArthurSampaio**
#### What is the purpose of this pull request?

Remove global handlers in order to use CSS modules.

#### How should this be manually tested?
https://shelfcssmodules--storecomponents.myvtex.com/

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/RelatedProducts.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ export default class RelatedProducts extends Component {
     }
 
     return (
-      <div className="vtex-related-products">
+      <div className={`${shelf.relatedProducts}`}>
```
Comment (Klynger) [inline react/RelatedProducts.js]: You don't need to use string interpolation here, you can just do `className={shelf.relatedProducts}`

File: react/Shelf.js [hunk 2]
```diff
@@ -35,7 +36,7 @@ class Shelf extends Component {
     }
 
     return (
-      <div className="vtex-shelf">
+      <div className={`${shelf.container}`}>
```
Comment (Klynger) [inline react/Shelf.js]: Same here

File: react/RelatedProducts.js (hunk 1 above)
Comment (ArthurSampaio) [reply react/RelatedProducts.js]: done

File: react/Shelf.js (hunk 2 above)
Comment (ArthurSampaio) [reply react/Shelf.js]: done

**Pull request merged**